### PR TITLE
Fix login as deleted user  and admin self delete bugs

### PR DESCRIPTION
**Pull request by IliaStepanov**

### CODE REVIEW

**Comment by IliaStepanov** [comment]
[[FIX] Deleted user can log  in + admin can delete himself](https://trello.com/c/xrsP180t/82-fix-deleted-user-can-log-in-admin-can-delete-himself)

File: src/main/java/com/epam/lowcost/controller/UserController.java [hunk 1]
```diff
@@ -125,10 +126,16 @@ public String changePassword(@ModelAttribute("sessionUser") User sessionUser, @R
...
-    public String deleteUser(@ModelAttribute(value = "sessionUser") User sessionUser, @RequestParam long id, Model model) {
+    public String deleteUser(@RequestParam long id, ModelMap model) {
+        User sessionUser = (User) model.get("sessionUser");
         if (!sessionUser.isAdmin()) {
             return "redirect:" + TICKETS + SELF;
         }
+        if (sessionUser.getId() == id) {
+            System.out.println(sessionUser.getId() + "" + id);
+            model.addAttribute("message", "You cant delete yourself!");
+            return "users";
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/controller/UserController.java:137]: In me opinion all names of views should be as constants too. single new task

File: src/main/java/com/epam/lowcost/service/implementations/UserServiceImpl.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public String deleteUser(long userId) {
     @Override
     public User verifyUser(String email, String password) {
         User user = userDAO.findByEmail(email);
-        if (user != null && BCrypt.checkpw(password, user.getPassword())) return user;
+        if (user != null && BCrypt.checkpw(password, user.getPassword()) && !user.isDeleted()) return user;
```
Comment (vakhonin) [inline src/main/java/com/epam/lowcost/service/implementations/UserServiceImpl.java]: `}{` in the all places

File: src/main/java/com/epam/lowcost/controller/UserController.java (hunk 1 above)
Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/controller/UserController.java:137]: Made a card in trello.

File: src/main/java/com/epam/lowcost/service/implementations/UserServiceImpl.java (hunk 2 above)
Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/service/implementations/UserServiceImpl.java]: Done.

**Pull request merged**